CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/time/LayerByTimePlugin.java

```diff
@@ -515,7 +517,7 @@ private void buildIntervals(final GraphWriteMethods wgcopy, final ArrayList<Floa
             // Only use transactions that have a datetime value set.
             final long date = wgcopy.getLongValue(dtAttr, txId);
 
-            if (date != ZonedDateTimeAttributeDescription.NULL_VALUE && d1t <= date && date < d2t) {
```

Comment:
This check is against the long value of a datetime. `ZonedDateTimeAttributeDescription.DEFAULT_VALUE` is a `ZonedDateTime` (and defaults to null), so the comparison doesn't make sense. I suspect it was a long at one point in time as the defaults for `LocalDateTimeAttributeDescription`, `DateAttributeDescription` and `TimeAttributeDescription` were all longs - however this would still be bad design as it assumes the user has chosen an attribute backed by `ZonedDateTimeAttributeDescription.ATTRIBUTE_TYPE` while still supporting attributes backed by other datetime types.